packages/dev-server/package.json

```diff
@@ -59,9 +60,13 @@
     "source-map": "^0.7.4",
     "ws": "^8.7.0"
   },
+  "peerDependencies": {
```

Comment:
Following up on our discussion
```suggestion
  "peerDependenciesMeta": {
```

Reply:
btw, will this satisfy pnpm too when sync importing `@react-native/dev-middleware`?

Reply:
we decided to include it as direct dep for now, we'll revisit this subject when the oldest supported RN version includes dev-middleware